gobblin-test-harness/src/main/java/gobblin/test/setup/config/Step.java

```diff
@@ -0,0 +1,31 @@
+package gobblin.test.setup.config;
+
+import java.util.Collection; 
+
+import gobblin.test.execution.operator.Operator;
+
+/**
+ * This interface is to define individual steps associated to a config entry, this implements the Operator interface.
+ * Essentially one step can have many operators and each operator has an execution
+ * 
+ * @author sveerama
+ *
+ */
+
+public interface Step extends Operator
```

Comment:
Will document . But from high level..

Read the config using ConfigReader which parses and validates the config. Then we create the steps for the config entry using the ConfigStepsGenerator.. Each step can have many setup operators.. For example, step is to getLsr, the 1st step is to look for the LSR files, if not there generate. There are 2 operators check for the lsr file and 2nd is to copy the input file and then generate the file status based on the input..